cloud/amazon/ecs_taskdefinition.py

```diff
@@ -142,12 +169,12 @@ def main():
 
     argument_spec = ec2_argument_spec()
     argument_spec.update(dict(
-        state=dict(required=True, choices=['present', 'absent'] ),
-        arn=dict(required=False, type='str' ),
-        family=dict(required=False, type='str' ),
-        revision=dict(required=False, type='int' ),
-        containers=dict(required=False, type='list' ),
-        volumes=dict(required=False, type='list' )
+        state=dict(required=True, choices=['present', 'absent']),
+        arn=dict(required=False, type='str'),
+        family=dict(required=False, type='str'),
+        revision=dict(required=False, type='int'),
+        containers=dict(required=False, type='list'),
+        volumes=dict(required=False, type='list')
```

Comment:
Good point @wimnat, I only suggested the changes because of the amount of time it takes for changes to be merged. We can definitely merge this in and  wait for the other changes as well. I'll be happy to help with reviews when changes are made.